### ISSUE TITLE
Does the trainning data only need HR images?

### ISSUE THREAD

**Issue by Macro03** [LongguangWang/DASR#8]
Hello, thank you very much for your excellent work. Does the trainning data only need HR images? Can I delete the self.dir_lr in df2k.py ?

**Comment by LongguangWang** [comment]
Hi @Macro03, thanks for your interest in our work.
In our implementation, the LR images are generated online during training thus you can safely remove `self.dir_lr`.

**Comment by Macro03** [comment]
I trained the Moco by the two real world images instead of 'degrade' images, the contrast_loss increase in early epoch and then decrease slowly. What do you think of it? Thank you so munch!

**Comment by LongguangWang** [comment]
Hi @Macro03, we also have this observation in our experiments on synthetic images. In my opinion, at early epochs, the [random samples](https://github.com/LongguangWang/DASR/blob/main/moco/builder.py#L34) in the queue is gradually replaced with the [training data](https://github.com/LongguangWang/DASR/blob/main/moco/builder.py#L146). Thus, it becomes more difficult to distinguish positive samples from negative samples and the loss is increased. After a few epochs, the queue is completely replaced by the training data and the loss begins to decrease.
